From: Steve Best <sbest@redhat.com>
Date: Mon, 30 Aug 2010 15:16:58 -0400
Subject: [scsi] ipr: fix compile warning
Message-id: <20100830150511.5212.35311.sendpatchset@squad5-lp1.lab.bos.redhat.com>
Patchwork-id: 27915
O-Subject: [PATCH RHEL5.6 BZ626566 1/16] Fix a compilation warning for the RHEL
	5 build
Bugzilla: 626566

RHBZ#:
------
https://bugzilla.redhat.com/show_bug.cgi?id=626566

Description:
------------
fix for a compilation warning for patch #4 submitted in
RHBZ bug 563589

RHEL Version Found:
-------------------
5.6

Brew:
-----
Built on all platforms. All patches brewed together.
http://brewweb.devel.redhat.com/brew/taskinfo?taskID=2713332

Upstream Status:
----------------
This change is only needed for RHEL 5.6
---------------------------------------

Steve Best
IBM on-site partner

Proposed Patch:
---------------

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/ipr.c b/drivers/scsi/ipr.c
index b1aa814..f9a23a2 100644
--- a/drivers/scsi/ipr.c
+++ b/drivers/scsi/ipr.c
@@ -8282,7 +8282,7 @@ ipr_get_chip_info(const struct pci_device_id *dev_id)
  * Return value:
  * 	0 on success / non-zero on failure
  **/
-static irqreturn_t ipr_test_intr(int irq, void *devp)
+static irqreturn_t ipr_test_intr(int irq, void *devp, struct pt_regs *regs)
 {
 	struct ipr_ioa_cfg *ioa_cfg = (struct ipr_ioa_cfg *)devp;
 	unsigned long lock_flags = 0;
